src/core/unplugin.ts

```diff
@@ -13,7 +13,7 @@ const PLUGIN_NAME = 'unplugin:webpack'
 
 export default createUnplugin<Options>((options = {}) => {
   const filter = createFilter(
-    options.include || [/\.vue$/, /\.vue\?vue/, /\.vue\?v=/],
+    options.include || [/\.vue$/, /\.vue(\.[t|j]sx?)?\?vue/, /\.vue\?v=/],
```

Comment:
The `|` can be removed:

https://regexper.com/#%2F%5C.vue%28%5C.%5Btj%5Dsx%3F%29%3F%5C%3Fvue%2F

![image](https://github.com/user-attachments/assets/0afad59b-7e0e-4897-85c2-49e7f89e60ca)


```suggestion
    options.include || [/\.vue$/, /\.vue(\.[tj]sx?)?\?vue/, /\.vue\?v=/],
```